src/Servers/Kestrel/test/BindTests/AddressRegistrationTests.cs

```diff
@@ -34,8 +34,7 @@ public class AddressRegistrationTests : TestApplicationErrorLoggerLoggedTest
         private const int MaxRetries = 10;
 
         [ConditionalFact]
-        [HostNameIsReachable]
```

Comment:
Do we still need HostNameIsReachable for other platforms?

Reply:
No, I think we can remove it now.